src/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { ThemeProvider } from 'styled-components';
+import { theme } from './styles/theme';
+import GlobalStyle from './styles/glabal';
+import Router from './Router';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <GlobalStyle>
+      <ThemeProvider theme={theme}>
+        <Router />
+      </ThemeProvider>
+    </GlobalStyle>
+  </React.StrictMode>,
```

Comment:
❗ 글로벌 스타일 적용 문법이 잘못되었습니다.
```suggestion
  <React.StrictMode>
    <ThemeProvider theme={theme}>
      <GlobalStyle />
      <Router />
    </ThemeProvider>
  </React.StrictMode>,
```